tasks/21_check-letters/check-letters.js

```diff
@@ -0,0 +1,17 @@
+// Make an empty array, run indexOf() on first element of input array.
+// Make sure that case is ignored adding toLowerCase() and fill output
+// array with number of indices. Run array every method to find out
+// if all elements in output array > -1,(exist in input string)
+
+export const checkLetters = ( arr ) =>{
+  const result = [];
+  const lowercased = arr[0].toLowerCase();
+
+  for ( let item of arr[1].toLowerCase() ) {
+    result.push( lowercased.indexOf(item) );
```

Comment:
You can do skip the last check with every and not use result array for it. For it you can just check in array and if `indexOf(item) === -1` then `return falsel` in the end of function `return true;`

Reply:
Refactored